app/src/main/java/org/kiwix/kiwixmobile/utils/KiwixTextToSpeech.java

```diff
@@ -109,9 +120,19 @@ public void readAloud(WebView webView) {
       } else {
         tts.setLanguage(locale);
 
-        // We use JavaScript to get the content of the page conveniently, earlier making some
-        // changes in the page
-        webView.loadUrl("javascript:" +
+        if(requestAudioFocus() == true) {
```

Comment:
Google Java style guide. spacing. Please check this as it is an issue in multiple locations.